apps/earn-protocol/components/organisms/Charts/components/ForecastTooltip.tsx

```
@@ -0,0 +1,150 @@
+export const ForecastTooltip = ({
+  active,
+  payload,
+  label,
+  tokenPrice,
+}: TooltipProps<number | [number, number], 'bounds' | 'forecast'> & {
+  tokenPrice?: string | null
+}) => {
+  const parsedPayload = payload?.reduce(
+    (acc, { name: valueName, value }) => {
+      return {
+        ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)